test/acceptance/features/steps/steps.py

```diff
@@ -303,24 +307,34 @@ def quarkus_app_is_imported_as_knative_service(context, application_name):
 
 
 # STEP
-
-
 @then(u'"{app_name}" deployment must contain SBR name "{sbr_name}"')
 def then_envFrom_contains(context, app_name, sbr_name):
     time.sleep(60)
     openshift = Openshift()
+    secret = ""
+    sbr = ServiceBinding()
+    if "sbr_name" in context:
+        json_path = "{.status.secret}"
+        secret = sbr.get_servicebinding_info_by_jsonpath(context.sbr_name, context.namespace.name, json_path)
```

Comment:
We should poll here unti we really get the value. `status.secret` might not exist at this point.

Reply:
Done here b7139c7